assets/panel/debugger.properties

```diff
@@ -439,6 +439,9 @@ original=original
 # LOCALIZATION NOTE (expressions.placeholder): Placeholder text for expression
 # input element
 expressions.placeholder=Add watch expression
+# LOCALIZATION NOTE (expressions.placeholder): Placeholder text for expression
```

Comment:
Comments are not mandatory, but if you add one:
* Update the text to match the new string, don't simply copy over an existing comment. This string is an error message, not a placeholder.
* Make sure that the string ID between parenthesis matches the actual string (`expressions.errorMsg`, not `expressions.placeholder`).

Reply:
@flodolo bullet 2 was an oversight, but I did bullet one on purpose as both are put in the placeholder attribute. But it makes sense to have it match its purpose.